src/fsm.jl

```diff
@@ -313,3 +313,28 @@ Return a minimal equivalent fsm.
 """
 minimize(fsm::FSM{T}) where T = (transpose ∘ determinize ∘ transpose ∘ determinize)(fsm)
 
+"""
+    union(fsm1, fsm2, ...)
+
+Merge several FSMs into a single one.
+"""
+function Base.union(fsm1::FSM{T}, fsm2::FSM{T}) where T
+    fsm = FSM{T}()
+    s = addstate!(fsm)
+    s1 = addstate!(fsm, label = "#fsm1")
+    s2 = addstate!(fsm, label = "#fsm2")
+    link!(fsm, s, s1)
+    link!(fsm, s, s2)
+    setinit!(s)
```

Comment:
You don't need the 's' state, you just have to mark `s1` and `s2` as initial state.

Reply:
I see.